.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: rust
-rust: nightly
-script: cd test-project && cargo test
+rust:
+ - stable
+ - nightly
+matrix:
+ include:
+   - rust: stable
+     env: FEATURES="--features stable"
+   - rust: nightly
+     env: FEATURES=""
+
+install: cargo build --verbose $FEATURES
```

Comment:
This line can be removed. `cargo test` on the next line will automatically build the library.

Reply:
OK, done.